rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/reports/TabulatorGenerator.xtend

```diff
@@ -228,7 +229,7 @@ class TabulatorGenerator {
 			if (functionOutputType instanceof RDataType) {
 				val context = createFunctionTabulatorContext(typeTranslator, func)
 				
-				val type = functionOutputType
+				val type = functionOutputType as RDataType
```

Comment:
Is this cast necessary? Why was it not there before?

Reply:
fixed, also I removed that storage into `type` as it's pointless